config/main.py

```diff
@@ -906,6 +906,34 @@ def speed(ctx, interface_name, interface_speed, verbose):
         command += " -vv"
     run_command(command, display_cmd=verbose)
 
+#
+# 'mtu' subcommand
+#
+
+@interface.command()
+@click.pass_context
+@click.argument('interface_name', metavar='<interface_name>', required=True)
+@click.argument('interface_mtu', metavar='<interface_mtu>', required=True)
+@click.option('-v', '--verbose', is_flag=True, help="Enable verbose output")
+def mtu(ctx, interface_name, interface_mtu, verbose):
+    """Set interface mtu"""
+    config_db = ctx.obj['config_db']
+    if get_interface_naming_mode() == "alias":
+        interface_name = interface_alias_to_name(interface_name)
+        if interface_name is None:
+            ctx.fail("'interface_name' is None!")
+
+    if interface_name_is_valid(interface_name) is False:
+        ctx.fail("Interface name is invalid. Please enter a valid interface name!!")
+
+    if interface_name.startswith("Ethernet"):
```

Comment:
SONiC port names are not guaranteed to begin with "Ethernet". This is not a reliable way to determine whether an interface is a front-panel port. There are examples in the `show` command (not sure if there are any yet in the `config` command) where this determination is made by checking which table in ConfigDB contains the interface name. Please refer to those examples for a proper solution.

Reply:
There is a certain truth in what you say. But I could not find such an example in the `show` command(/src/sonic-utilities/show/main.py). Can you give a specific code or a corresponding location? Many thanks.

Reply:
Sorry, I was mistaken. The example I was thinking of is in the intfutil utility. Here's a direct link to a line which is checking whether or not an interface is a front-panel interface by name: https://github.com/Azure/sonic-utilities/blob/master/scripts/intfutil#L67